spec/features/accept_organization_invitation_spec.rb

```diff
@@ -58,8 +57,10 @@
         visit accept_invitation_url(invitation, token: invitation.token)
       end
 
-      it { expect(page).to have_content "You joined #{admin_member.organization.name}" }
-      it { expect(page).to have_content "Sign out" }
+      it do
```

Comment:
@dissident I see above line `it { expect(page).to have_content "Sign in" }`. I believe it may be integrated too. Without integration it runs separate spec.